app/src/main/java/kr/ac/konkuk/gdsc/plantory/presentation/plant/AddPlantFragment.kt

```diff
@@ -57,31 +67,61 @@ class AddPlantFragment : BindingFragment<FragmentAddPlantBinding>(R.layout.fragm
     private val getContent =
         registerForActivityResult(ActivityResultContracts.GetContent()) { uri: Uri? ->
             if (uri != null) {
-                val newPlantInfo = viewModel.plantInfo.value.copy(image = uri)
-                viewModel.updatePlantInfo(newPlantInfo)
+                viewModel.updatePlantImage(uri)
+                updateRequestBody()
             }
         }
 
+    private fun initRegisterButtonClickListener() {
+        binding.btnAddplantUpload.setOnSingleClickListener {
+            viewModel.viewModelScope.launch {
+                viewModel.postRegisterPlant()
+            }
+        }
+    }
+
+    private fun setPostRegisterPlantStateObserver() {
+        viewModel.postRegisterPlantState.flowWithLifecycle(viewLifeCycle).onEach { state ->
+            when (state) {
+                is UiState.Success -> {
+                    Timber.d("Success : Register ")
+                    parentFragmentManager.popBackStack()
+                }
+
+                is UiState.Failure -> {
+                    Timber.d("Failure : ${state.msg}")
```

Comment:
```suggestion
                    Timber.e("Failure : ${state.msg}")
```

error가 난 경우에는 의미론상 e를 사용하는게 더 좋을 것 같습니다!